odbc/src/api/functions.rs

```diff
@@ -1785,11 +1785,441 @@ pub unsafe extern "C" fn SQLGetInfo(
 #[no_mangle]
 pub unsafe extern "C" fn SQLGetInfoW(
     connection_handle: HDbc,
-    _info_type: InfoType,
-    _info_value_ptr: Pointer,
-    _buffer_length: SmallInt,
-    _string_length_ptr: *mut SmallInt,
+    info_type: InfoType,
+    info_value_ptr: Pointer,
+    buffer_length: SmallInt,
+    string_length_ptr: *mut SmallInt,
+) -> SqlReturn {
+    panic_safe_exec!(
+        || sql_get_infow_helper(
+            connection_handle,
+            info_type,
+            info_value_ptr,
+            buffer_length,
+            string_length_ptr
+        ),
+        connection_handle
+    )
+}
+
+unsafe fn sql_get_infow_helper(
+    connection_handle: HDbc,
+    info_type: InfoType,
+    info_value_ptr: Pointer,
+    buffer_length: SmallInt,
+    string_length_ptr: *mut SmallInt,
 ) -> SqlReturn {
+    match info_type {
+        // SQL_DRIVER_NAME
+        InfoType::DriverName => {
+            // A character string with the file name of the driver used to access the data source.
+            // TODO: (code review) What file name should be used here? Is this something that
+            //       will need to be dynamically determined or is it something we can hardcode? 
```

Comment:
This is the style of comment I'm talking about: in these `TODO: (code review)` comments, I added some info about the `InfoType` and asked questions. I appreciate any help!

Reply:
I checked with Alexi.
To match the JDBC driver, she wants to use : MongoDB Atlas SQL interface ODBC Driver